pkg/clockcache/cache.go

```diff
@@ -288,6 +299,20 @@ func (self *Cache) Len() int {
 	return len(self.storage)
 }
 
+func (self *Cache) Evictions() uint64 {
+	self.elementsLock.RLock()
+	defer self.elementsLock.RUnlock()
+	return self.evictions
+}
+
+func (self *Cache) SizeBytes() uint64 {
+	self.elementsLock.RLock()
+	defer self.elementsLock.RUnlock()
+	//derived from BenchmarkMemoryEmptyCache 120 bytes per element
+	cacheSize := cap(self.storage) * 120
+	return uint64(cacheSize) + self.dataSize
```

Comment:
Aren't we overcalculating here? The cap already has the `dataSize` in itself, so the addition of that is not clear to me. Do we have any tests to make sure about this? (as unit tests, not benchmarks, so that its easier to review)

Reply:
No, the element has only a pointer (interface{} is a pointer) to the key and value 
```
type element struct {
	// The value stored with this element.
	key   interface{}
	value interface{}
...
```
so the data size is in addition to the size of the cache itself